app/views/sessions/new.html.erb

```diff
@@ -1,51 +1,36 @@
 <% content_for :head do %>
   <%= javascript_include_tag "login" %>
+  <%= javascript_include_tag "auth_providers" %>
 <% end %>
 
-<% content_for :heading do %>
-  <h1><%= t ".heading" %></h1>
-<% end %>
-
-<div id="login_login">
-  <p class='text-muted'><%= t ".no account" %> <%= link_to t(".register now"), user_new_path(:referer => params[:referer]) %></p>
-
-  <%= bootstrap_form_tag(:action => "login", :html => { :id => "login_form" }) do |f| %>
-    <%= hidden_field_tag("referer", h(params[:referer]), :autocomplete => "off") %>
-
-    <%= f.text_field :username, :label => t(".email or username"), :tabindex => 1, :value => params[:username] %>
-    <%= f.password_field :password, :label => t(".password"), :tabindex => 2, :value => "", :help => link_to(t(".lost password link"), user_forgot_password_path) %>
-    <%= f.form_group do %>
-      <%= f.check_box :remember_me, { :label => t(".remember"), :tabindex => 3, :checked => (params[:remember_me] == "yes") }, "yes" %>
-    <% end %>
+<div id="login_login" class="auth-container">
+  <div class="tabs">
+    <button class="tab-button active" disabled="true"><%= t ".tab_title" %></button>
+    <button class="tab-button" onclick="location.pathname='<%= url_for(:action => :new, :controller => :users) -%>'"><%= t "users.new.tab_title" %></button>
```

Comment:
Why onclick? Why button? Why isn't it a link?

https://getbootstrap.com/docs/5.3/components/navs-tabs/#tabs

Reply:
The `onlick` won't work anyway as our security policy doesn't allow inline javascript.

Reply:
Fixed to use links instead.